keyboards/qck75/v1/rules.mk

```
@@ -0,0 +1,5 @@
+# This file intentionally left blank
+NKRO_ENABLE = yes
+OLED_ENABLE = yes
+OLED_DRIVER = SSD1306
+WPM_ENABLE = yes
```

File isnt blank so comment doesnt stand
```suggestion
OLED_DRIVER = SSD1306
```